includes/api/class-wc-rest-products-controller.php

```diff
@@ -134,6 +134,31 @@ public function query_args( $args, $request ) {
 			'product_shipping_class' => 'shipping_class',
 		);
 
+		/*
+		 * TO-DO: Figure out a better way to get all taxonomies available
+		 * to a custom post type than grabbing an entire post.
+		*/
+
+		// Set up our temporary arguments to grab a product post.
+		$tmp_args = array(
+			'post_type' => 'products',
+			'posts_per_page' => 1,
+		);
+
+		// Grab the first product.
+		$tmp_product = get_posts( $args )[0];
```

Comment:
This is the line that failed Travis checks for PHP 5.2 and 5.3

`PHP Parse error:  syntax error, unexpected '['`